__init__.py

```diff
@@ -102,24 +106,84 @@ def attempt_three_times(function):
                 raise
             msg = "Caught {error}: {message}. Retrying...".format(
                 error=type(e), message=e.message)
-            logger.warning(msg)
+            LOG.warning(msg)
+
+
+#######################################################################
```

Comment:
This commenting style is inconsistent with the rest of the file. In addition, there is no indication of where these helpers end. The divider makes it seem that the section is composed of everything below it.